mycroft/skills/core.py

```diff
@@ -1648,6 +1650,26 @@ def cancel_all_repeating_events(self):
         for e in list(self.scheduled_repeats):
             self.cancel_scheduled_event(e)
 
+    def acknowledge(self):
+        """ Acknowledge a successful request.
+
+        This method plays a sound to acknowledge a request that does not
+        require a verbal response. This is intended to provide simple feedback
+        to the user that their request was handled successfully.
+        """
+        file = resolve_resource_file(
```

Comment:
nit: I would discourage use of "file" as a variable name. While `file` is no longer a keyword in python3, most syntax highlighters are going to make it look risky (github's included!). Thanks!

Reply:
Good catch. I agree using file is not ideal. I copied from https://github.com/MycroftAI/mycroft-core/blob/dev/mycroft/client/speech/mic.py#L513. I'll change it in both places.